Source/ACE/Entity/Landblock.cs

```diff
@@ -369,22 +408,27 @@ public void UseTime()
             {
                 // here we'd move server objects in motion (subject to landscape) and do physics collision detection
 
-                // for now, we'll move players around
-                List<WorldObject> movedObjects = null;
-                List<Player> players = null;
+                // for now, we'll move players around, and stream static objects
+                List<WorldObject> allworldobj = null;
+                List<Player> allplayers = null;
+                List<Player> movedObjects = null;
                 List<WorldObject> despawnObjects = null;
                 List<Creature> deadCreatures = null;
 
                 lock (objectCacheLocker)
                 {
-                    movedObjects = this.worldObjects.Values.OfType<WorldObject>().ToList();
-                    players = this.worldObjects.Values.OfType<Player>().ToList();
-                    despawnObjects = this.worldObjects.Values.ToList();
-                    deadCreatures = this.worldObjects.Values.OfType<Creature>().ToList();
+                    allworldobj = this.worldObjects.Values.OfType<WorldObject>().ToList();
                 }

```

Comment:
Yea, I missed that one.
I use to need to cast them to their types but then I changed it to lock in all world objects instead of each one, since I now need to know all of them and not just players, etc

Will fix tomorrow